templates/cloudformation/iam_user_osdCcsAdmin.json

```diff
@@ -6,8 +6,7 @@
         "ManagedPolicyArns": [
           "arn:aws:iam::aws:policy/AdministratorAccess"
         ],
-        "UserName": "osdCcsAdmin",
-        "Tags" : [{"Key": "MOA", "Value": "true"}]
```

Comment:
Did you intend to remove these?

Reply:
I don't think we ever used these. I can bring them back if they are useful for something else, or we could rename them to `ROSA`, but I don't find references to it in our code

Reply:
I remembered them being talked about somewhere it was [here](https://github.com/openshift/moactl/pull/151#discussion_r507683779) - Looks like they are not being used.

Reply:
It is not working as of right now; I have an open issue in this regard on aws-go-sdk.
When I will make this work, I will open a new PR.